tests/dataset/conftest.py

```diff
@@ -267,12 +271,15 @@ def multi_dataset(experiment, request: FixtureRequest):
         datasaver.dataset.conn.close()
 
 
-@pytest.fixture(scope="function", params=["array"])
+@pytest.fixture(scope="function", params=[True, False])
```

Comment:
does this reflect cleanly in the name of the test when the summary of the execution is shown? I'd think that adding pytest.parameterize with a clear parameter name, e.g. "is_shape_set_explicitly", would improve the readability of the test summary and the test itself